core/build.gradle

```diff
@@ -57,4 +57,9 @@ dependencies {
     compile "io.reactivex:rxandroid:$rxAndroidVersion"
 
     compile "com.github.AntennaPod:AntennaPod-AudioPlayer:$audioPlayerVersion"
+
+    // Add casting features
+    compile "com.google.android.libraries.cast.companionlibrary:ccl:$castCompanionLibVer"
+    //compile "com.android.support:mediarouter-v7:$supportVersion"
+    //compile "com.google.android.gms:play-services:$playServicesVersion"
```

Comment:
Either sounds fine to me. Which one would you prefer?
We're using both libraries directly at some point, although I don't think ccl will ever stop using them either.
In the long run, we might just do it all without ccl, as it is a nice for a basic chromecast app, but for us much of what if offers is just redundant.